core/src/main/java/org/apache/brooklyn/util/core/ResourcePredicates.java

```diff
@@ -64,7 +66,7 @@ public boolean apply(@Nullable String resource, @Nullable BrooklynObject context
 
         @Override
         public String toString() {
-            return "ResourcePredicates.exists()";
+            return "ResourcePredicates.urlExists()";
```

Comment:
Shouldn't this be simply `urlExist()`? Would be more consistent with the other predicate introduced by this PR

Reply:
i don't follow -- why would `urlExist` be better?